plugin.video.catchuptvandmore/resources/lib/websites/lesargonautes.py

```diff
@@ -0,0 +1,126 @@
+                                                                                                    
+                                                                                                                            item = Listitem()
+                                                                                                                            item.label = video_title
+                                                                                                                                item.art['thumb'] = video_image
+                                                                                                                                item.info['plot'] = video_plot
+                                                                                                                            
+                                                                                                                                item.context.script(
+                                                                                                                                                    get_video_url,
+                                                                                                                                                    plugin.localize(LABELS['Download']),
+                                                                                                                                                    item_id=item_id,
+                                                                                                                                                    video_id=video_id,
+                                                                                                                                                    video_label=LABELS[item_id] + ' - ' + item.label,
+                                                                                                                                                    download_mode=True)
+                                                                                                                                    
+                                                                                                                                    item.set_callback(
+                                                                                                                                                      get_video_url,
+                                                                                                                                                      item_id=item_id,
+                                                                                                                                                      video_id=video_id)
+                                                                                                                                        yield item
```

Comment:
Indentation in the above two functions is completely wrong, this isn't valid python.

Reply:
Oups, thank you.
Correction done